### PR TITLE
DQt2: Build on OS X

### PR DESCRIPTION
**Pull request by jaglo**
Modified DQt files to enable successful DQt2 build on OS X.

Included Mac icon file in DQt2 Resources directory.

---

This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/3342)

### CODE REVIEW

**Comment by BhaaLseN** [comment]
@dolphin-emu-bot rebuild

File: Source/Core/DolphinQt2/MainWindow.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ MainWindow::MainWindow() : QMainWindow(nullptr)
 {
 	setWindowTitle(tr("Dolphin"));
 	setWindowIcon(QIcon(Resources::GetMisc(Resources::LOGO_SMALL)));
-
+	this->setUnifiedTitleAndToolBarOnMac(true);
```
Comment (spxtr) [inline Source/Core/DolphinQt2/MainWindow.cpp]: You don't need `this->`.

**Comment by RisingFog** [comment]
This PR needs to be rebased.

**Comment by turbedi** [comment]
This PR is obsolete since PR #3817 has been merged.

**Pull request closed without merging**